### DISCUSSION TITLE
Unselect All check box for Update Manager in Mint

### DISCUSSION THREAD

**Discussion by motoryzen** [linuxmint/discussions#625] [Ideas]
I don't need to update everything, and thus often I'm having to painstakenly mouse over and uncheck so many items it just kills a lot of time to only update a few things I need and care about.  

There is already a " select all " button. Now please make a " Unselect All " check button please.  Thanks dev team(s)

**Comment by mtwebster** [comment]
Look to the left of the "Select all" button - 
![image](https://github.com/user-attachments/assets/ffbb6287-42e7-4fd7-8434-04301da47da6)

**Reply by motoryzen** [reply]
*makes nervous chuckle*

Cool.  There's my sign.    Thanks
